mytoyota/client.py

```diff
@@ -102,12 +102,7 @@ def get_supported_regions() -> list:
         Returns:
             A list of supported regions. For example: ["europe"]
         """
-        regions = []
-
-        for key, _ in SUPPORTED_REGIONS.items():  # pylint: disable=unused-variable
-            regions.append(key)
-
-        return regions
+        return [key for key in SUPPORTED_REGIONS.keys()]

```

Comment:
Better replace identity comprehension with call to collection constructore here:
`return list(SUPPORTED_REGIONS.keys())`